docs/pbc/all/tax-management/tax-management.md

```diff
@@ -19,13 +19,14 @@ redirect_from:
   - /docs/international-tax-rates-sets-1
   - /docs/en/international-tax-rates-sets-1
   - /docs/scos/user/features/202200.0/tax-feature-overview.html
+  - /docs/scos/user/features/202204.0/tax-feature-overview.html  
 ---
 
 The *Tax* feature lets you define taxes for the items you sell. The feature is represented by two entities: tax rates and tax sets.
```

Comment:
We need to replace `Tax feature` with `Tax Management PBC` or `Tax Management capability`.